### DISCUSSION TITLE
AffableBean project Snapshot Unavailable. Can you please check?

### DISCUSSION THREAD

**Discussion by lcgummad** [apache/netbeans#4408] [General]
### Apache NetBeans version

Apache NetBeans 14

### What happened

https://netbeans.apache.org/kb/docs/javaee/ecommerce/conclusion.html

In the above tutorial, project Snapshot links are not working. I get "Page not Found" error. For example,

In https://netbeans.apache.org/kb/docs/javaee/ecommerce/entity-session.html

AffableBean project | snapshot 3

link gives "Page not Found" error.

where can i find the project snapshot that i can directly upload into netbeans and run the project instead of coding the pages myself

Thanks,

### How to reproduce

Go to 
In https://netbeans.apache.org/kb/docs/javaee/ecommerce/entity-session.html

and click on 

AffableBean project | snapshot 3

"Page not Found" 

### Did this work correctly in an earlier version?

Apache NetBeans 12.3 or earlier

### Operating System

Windows 10 Home

### JDK

JDK 13.0.1 (64-BIT)

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

Yes

### Code of Conduct

Yes

**Comment by ebarboni** [comment]
this is more a website issue @neilcsmith-net @geertjanw could we "open" the issue tab on the netbeans-website to make a transfert ?

**Comment by neilcsmith-net** [comment]
Let's move this to discussions - either we open issues there or have a docs category in the discussions here for this?

**Reply by mbien** [reply]
sounds good (either of the two options).

Btw do those "How to Contribute" banners still exist or did I click it away and forgot about it? It could be next to the "Github issues migration" banner and link to the issue labels and one of the (many) contribution guidelines. It should mention the website too.

**Comment by mbien** [comment]
we could check the code into a (old-)netbeans-tutorials project on github - assuming it still exists somewhere.
